src/gui/EditWidgetIcons.cpp

```diff
@@ -142,6 +148,8 @@ void EditWidgetIcons::load(const QUuid& currentUuid,
             m_ui->defaultIconsRadio->setChecked(true);
         }
     }
+
+    m_ui->applyRecursivelyCheckBox->setChecked(false);
```

Comment:
Instead of a checkbox, I think it would be better to have a button labeled "Apply to Children". The checkbox state is not saved with the group.

Reply:
Where would be a suitable place for this button in your opinion?

The functionality appears to be too specific compared to the other buttons for placing it p.e. in the main menu or the toolbar. In EditWidgetIcons, the buttons seems to be out of place, because the user expects changes to be made only after clicking the OK button.

Reply:
Oh I really like the idea of the submenu button! The icon panel is strange. Technically when you download a favicon, for example, it modifies the database immediately. Even if you don't end up accepting the icon for the entry. I'll think about this, but you are correct.